llvm/include/llvm/CodeGen/ValueTypes.h

```diff
@@ -488,8 +488,8 @@ namespace llvm {
     Type *getTypeForEVT(LLVMContext &Context) const;
 
     /// Return the value type corresponding to the specified type.
-    /// This returns all pointers as iPTR.  If HandleUnknown is true, unknown
-    /// types are returned as Other, otherwise they are invalid.
+    /// If HandleUnknown is true, unknown types are returned as Other,
+    /// otherwise they are invalid.
```

Comment:
Maybe should make a note about pointer handling?

Reply:
Sure. I should also add to the comment in MachineValueTypes.td like used to be in the .h.

Reply:
Comments updated, including ValueTypes.td itself